src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecatation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
+- Private Registry credentials defined in helm has been deprecated. Private Registry Credentials are now supported natively by Okteto and credentials defined in helm will be automatically migrated. More info [here](ADD_LINK)
+
+### Bug Fixes
+
+- Reduce file permission for okteto-ca-pemstore certificate file in jobs  <!-- 7303 -->
+- Reduce expiration threshold for trial licenses from 60 to 7 days <!-- 7314 -->
```

Comment:
This might be misleading. The first time I read it I understood that the trials duration is now 7 days.

What about something like: `Warning message for licenses close to expire is displayed now 7 days before expiration instead of 60 for trial licenses`?

cc @omnomagonz

Reply:
If we are gonna have the scope of the change be just about the warning I would just skip the entry completely.